README.rst

```diff
@@ -122,15 +121,12 @@ on the target architecture.
 Method 2: pip installation
 --------------------------
 
-The Python package is directly installable via ``pip install Corrfunc``. However, in that case you will lose the ability to recompile the code according to your needs.  This usually fine for a single-machine installation, like a laptop, where you are only using the Python interface.  For usage on a cluster or other environment with multiple CPU architectures, you may find it more useful to use the source installation method above in case you need to compile for a different architecture later.
+The Python package is directly installable via ``python -m pip install Corrfunc``. However, in that case you will lose the ability to recompile the code.  This usually fine if you are only using the Python interface and are on a single machine, like a laptop.  For usage on a cluster or other environment with multiple CPU architectures, you may find it more useful to use the source installation method above in case you need to compile for a different architecture later.
 
-You can test a pip-installed Corrfunc with:
+You can check that a pip-installed Corrfunc is working with:
 
 ::
-   $ pip install pytest
-   $ pytest --pyargs Corrfunc
-
-The tests may take a few minutes to run.
+   $ python -m Corrfunc.tests
```

Comment:
Should this be `python -m pytest Corrfunc`?

Reply:
We're not actually distributing the full pytest suite as part of the pip package, so pytest doesn't really do anything.  But one can still use `python -m Corrfunc.tests` to invoke the simpler Python tests.